app/Repositories/Dislike/DislikeRepository.php

```diff
@@ -3,6 +3,7 @@
 namespace Hedonist\Repositories\Dislike;
 
 use Hedonist\Entities\Dislike\Dislike;
+use Hedonist\Entities\Place\Place;
```

Comment:
How it is possible? Place model doesn't exist in dev branch for now.

Reply:
i need to merge place branch to my?

Reply:
Not for now. Your code is ok, we will merge this pr after place model ready.